src/managers/storageManagers.ts

```diff
@@ -0,0 +1,28 @@
+import { notifyFavoriteChange } from "./eventManager/favoriteEventManager";
+
+export function storeFoodItems(foodItems: FoodItemType[]) {
+  localStorage.setItem("foodItems", JSON.stringify(foodItems));
+}
+
+export function getStoredFoodItems() {
+  const storedItems = localStorage.getItem("foodItems");
+  if (storedItems) return JSON.parse(storedItems);
+
+  return [];
+}
```

Comment:
```suggestion
export function getStoredFoodItems() {
  const storedItems = localStorage.getItem("foodItems");

  return storedItems ? JSON.parse(storedItems) : [];
}
```

[nit] 스타일 따라 다르겠지만, 이정도 분기는 삼항연산자를 활용하면 더 깔끔해질것 같아요

Reply:
우테코를 시작하고 삼항 연산자를 의식적으로 피하다 보니 간단한 조건에서도 if문을 작성했었네요 😅 
조건이 복잡하거나 / 중첩 If 등이 아닌 상황이라면 삼항 연산자를 적절하게 사용해 보겠습니다!

Reply:
개인적인 취향에 맞게 선택하셔요 ㅎㅎ